zanata-war/src/main/java/org/zanata/action/VersionHome.java

```diff
@@ -370,7 +370,6 @@ public String createVersion() {
     }
 
     public void copyVersion() {
-        getInstance().setStatus(EntityStatus.READONLY);
```

Comment:
I think to minimise the changes of security rules, its best that I restore the readonly status.